iidm/iidm-extensions/src/main/java/com/powsybl/iidm/network/extensions/ActivePowerControl.java

```diff
@@ -59,22 +59,22 @@ default String getName() {
     /**
      * @return if present, provides the overridden value of minP to be used for active power control operations.
      */
-    OptionalDouble getMinPOverride();
+    OptionalDouble getMinTargetP();
 
     /**
      * Sets the overridden minimal active power.
-     * @param pMinOverride  The overridden value of minP. A Nan value removes the override.
+     * @param minTargetP  The overridden value of minP. A Nan value removes the override.
      */
-    void setMinPOverride(double pMinOverride);
+    void setMinTargetP(double minTargetP);
```

Comment:
Can you add in the 2 setters' javadoc that the value should be in `[pMin, pMax]` ?

Reply:
Done